File: php/src/testEncryptHelper.php
<?php
include 'EncryptHelper.php';

$helper = new EncryptHelper();
$textToEncrypt = "将要加密的字符";
$key = "5dd69f95-a82a-4b01-b6c1-a2d2f8f8061b";
$encryptedText = "neD0iV1eFLpIy4/F1XHFEomEfx2kV/EPMg+edO9TU/8=";


if (strcmp($helper->encrypt($textToEncrypt, $key), $encryptedText) != 0) {
    throw new Exception('found error when encode string');
}
if (strcmp($helper->decrypt($encryptedText, $key), $textToEncrypt) != 0) {
    throw new Exception('found error when decode string');
}
?>

File: ruby/test/testEncryptHelper.rb
# encoding: UTF-8
require 'test/unit'
require_relative "../src/encryptHelper"

$textToEncrypt = "将要加密的字符"
$key ="5dd69f95-a82a-4b01-b6c1-a2d2f8f8061b"
$encryptedText = "neD0iV1eFLpIy4/F1XHFEomEfx2kV/EPMg+edO9TU/8="

class TestEncryptHelper < Test::Unit::TestCase

  def test_encrypt
    assert_equal($encryptedText,EncryptHelper.encrypt($textToEncrypt,$key))
  end

  def test_decrypt
    assert_equal($textToEncrypt,EncryptHelper.decrypt($encryptedText,$key))
  end
end